### ISSUE TITLE
Failed to build apex in cuda10.1 with newly updated code

### ISSUE THREAD

**Issue by Hanqer** [NVIDIA/apex#1090]
I use a docker environment to setup apex. The CUDA version is 10.1 with cudnn7. And the GPU is Tesla P100. 
When build apex with main (master) branch , it failed. The main error is some undefined identifiers.
`/tmp/pip-req-build-8tsq5zmy/csrc/mlp_cuda.cu(194): error: identifier "cublasLtMatmulDescOpaque_t" is undefined

    /tmp/pip-req-build-8tsq5zmy/csrc/mlp_cuda.cu(195): error: identifier "cublasLtMatrixLayoutOpaque_t" is undefined

    /tmp/pip-req-build-8tsq5zmy/csrc/mlp_cuda.cu(196): error: identifier "cublasLtMatmulPreferenceOpaque_t" is undefined

    /tmp/pip-req-build-8tsq5zmy/csrc/mlp_cuda.cu(205): error: identifier "CUBLAS_COMPUTE_32F" is undefined

    /tmp/pip-req-build-8tsq5zmy/csrc/mlp_cuda.cu(205): error: identifier "cublasLtMatmulDescInit" is undefined

    /tmp/pip-req-build-8tsq5zmy/csrc/mlp_cuda.cu(234): error: identifier "cublasLtMatrixLayoutInit" is undefined

    /tmp/pip-req-build-8tsq5zmy/csrc/mlp_cuda.cu(248): error: identifier "cublasLtMatmulPreferenceInit" is undefined

    /tmp/pip-req-build-8tsq5zmy/csrc/mlp_cuda.cu(336): error: identifier "cublasLtMatmulDescOpaque_t" is undefined

    /tmp/pip-req-build-8tsq5zmy/csrc/mlp_cuda.cu(337): error: identifier "cublasLtMatrixLayoutOpaque_t" is undefined

    /tmp/pip-req-build-8tsq5zmy/csrc/mlp_cuda.cu(338): error: identifier "cublasLtMatmulPreferenceOpaque_t" is undefined

    /tmp/pip-req-build-8tsq5zmy/csrc/mlp_cuda.cu(347): error: identifier "CUBLAS_COMPUTE_32F" is undefined

    /tmp/pip-req-build-8tsq5zmy/csrc/mlp_cuda.cu(347): error: identifier "cublasLtMatmulDescInit" is undefined

    /tmp/pip-req-build-8tsq5zmy/csrc/mlp_cuda.cu(376): error: identifier "cublasLtMatrixLayoutInit" is undefined

    /tmp/pip-req-build-8tsq5zmy/csrc/mlp_cuda.cu(390): error: identifier "cublasLtMatmulPreferenceInit" is undefined

    14 errors detected in the compilation of "/tmp/tmpxft_00000471_00000000-6_mlp_cuda.cpp1.ii".
    ninja: build stopped: subcommand failed.`

But when I reset to the commit with 'e5f2f675de1c4e85e772065e550db55b13264265'  (15 April), it is no error occur. And everything run well. 
Are there some new updates cause this error?

**Comment by ishitamed19** [comment]
Yes, I am also getting the same issue when I use the latest commit. Reverting to the `e5f2f675de1c4e85e772065e550db55b13264265` commit solves.

**Comment by AlexanderVanEck** [comment]
To reproduce;

This was introduced in https://github.com/NVIDIA/apex/commit/b8be1bc7b663294a121194e51aeebad40c31d60e

Dockerfile:

```
FROM nvidia/cuda:10.2-cudnn7-devel-ubuntu18.04

RUN pip uninstall apex && pip install git+https://github.com/NVIDIA/apex--no-cache-dir --global-option="--cpp_ext" --global-option="--cuda_ext"
```

@seryilmaz any fixes you could recommend/make?

**Comment by ptrblck** [comment]
@seryilmaz could you please take a look at this?
